pyproject.toml

```diff
@@ -340,6 +340,7 @@ line-length = 100
 target-version = "py310"
 
 [tool.ruff.lint]
+ignore = ["S603"] # This rule does not make sense in tests.
```

Comment:
You're disabling it globally for all the things outside tests. It should probably be in `tests/ruff.toml` or something.